git.md

```diff
@@ -143,7 +143,7 @@ Whatever the language you are using on a FUN project, cooking a new release \(_e
 
 3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,
 
-4. Commit your changes with a standard message title like: `Bump release to 4.18.1`,
+4. Commit your changes with a structured message, add a title including the name of the release like: `Bump release to 4.18.1`, add to the body all the changes from the changelog concerned by this release,
```

Comment:
maybe paste here an example of release commit?

Reply:
I've added an exemple.